css/article.css

```diff
@@ -0,0 +1,85 @@
+nav {
+  box-sizing: border-box;
+}
+
+.nav-links-wrapper {
+  padding: 20px 0;
+}
+.nav-links:hover {
+  transition: 0.3s;
+  color: #ff8c00;
+}
+
+.article-wrapper {
+  display: flex;
+  flex-direction: column;
+  color: #333;
+  box-sizing: border-box;
+}
+
+.article-content {
+  order: 1;
+}
+
+.article-content-title {
+  font-size: 1.4rem;
+  font-weight: 400;
+}
+
+.article-links-wrapper {
+  order: 2;
+  display: flex;
+  flex-direction: column;
+}
+
+.article-links-heading {
+  font-size: 1.07rem;
+  font-weight: 600;
+  color: #000;
+  display: block;
+  border-bottom: 1px solid #dfdfdf;
+  text-transform: uppercase;
+  margin-bottom: 26px;
+  padding-bottom: 5px;
+}
+
+.article-links {
+  font-size: 1.07rem;
+  color: #666;
+  display: block;
+  border-bottom: 1px solid #f3f3f3;
+  text-transform: capitalize;
+  padding: 10px 0;
+}
+
+.article-links:hover {
+  transition: 0.3s;
```

Comment:
same
don't mark as resolved if it's not resolved :) check all other places where's leading zero and remove it